src/main/java/algorithms/orthogonalRangeSearching/twoDim/README.md

```diff
@@ -0,0 +1,68 @@
+# 2D Orthogonal Range Searching
+
+2D orthogonal range searching is a computational problem that involves efficiently answering queries about points 
+in a two-dimensional space that fall within a specified axis-aligned rectangular range (orthogonal range). In other 
+words, given a set of points in 2D, find me all the points that lie within a specific rectangle. 
+
+To do so, we can extend the idea of 1D range trees to the 2D context. 
+
+Strategy
+- Each node in the x-tree has a set of points in its subtree
+- Store a y-tree at each x-node containing all the points in the x-subtree
+
+![2DORS](../../../../../../docs/assets/images/2DORS.jpg)
+
+1. Build an x-tree using only x-coordinates. 
+   - This should be done in the exact same way as you would for a 1D range tree. 
+2. For every node in the x-tree, build a y-tree out of nodes in the x-subtree using only y-coordinates. 
```

Comment:
it would help to emphasize that the **same set of nodes** are being used. previously was on the x-coord, now on the y-coord

Reply:
edit: yeah first time reading i was looking at text. its clearer with the accompanying pic. but yep, if u can make it clearer without ebing verbose, it'd be gud too!